exonum/src/runtime/mod.rs

```diff
@@ -205,11 +226,11 @@ pub trait Runtime: Send + Debug + 'static {
     ///
     /// ['start_service`]: #start_service
     /// ['stop_service`]: #stop_service
-    fn configure_service(
+    fn initialize_service(
         &self,
         fork: &Fork,
-        descriptor: InstanceDescriptor,
-        parameters: Any,
+        instance: InstanceDescriptor,
+        parameters: Vec<u8>,
```

Comment:
Is format of `parameters` runtime-specific?

Reply:
Format of parameters is a service specific, but I suppose that I may write that parameters should be a correct protobuf message.